snuba/state/rate_limit.py

```diff
@@ -111,26 +116,60 @@ def rate_limit(
     now = time.time()
     bypass_rate_limit, rate_history_s = state.get_configs(
         [("bypass_rate_limit", 0), ("rate_history_sec", 3600)]
+        #                               ^ number of seconds we keep the timestamps
     )
     assert isinstance(rate_history_s, (int, float))
 
+    # we don't really do this but you *can* skip rate limit
     if bypass_rate_limit == 1:
         yield None
         return
 
     pipe = rds.pipeline(transaction=False)
-    pipe.zremrangebyscore(
-        bucket, "-inf", "({:f}".format(now - rate_history_s)
-    )  # cleanup
+    # cleanup old queries: remove everything between [-inf and now - rate_history_s]
```

Comment:
I don't know that you need this comment, the redis function name is self explanatory.